src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,95 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_SIZE = 0;
+    private static final double CONDITION_RESIZE = 0.75;
```

Comment:
```suggestion
    private static final double LOAD_FACTOR = 0.75;
```
в хешмапі ця змінна має таку назву

Reply:
+